CodeStarterCamp_Week5/Talent.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Talent.swift
+//  CodeStarterCamp_Week5
+//
+//  Created by minsong kim on 2023/03/20.
+//
+
+import Foundation
+
+protocol Talent {
+    var singing: Level { get }
+    var dancing: Level { get }
+    var acting: Level { get }
+    var visual: Level { get }
+}
+
+extension Talent {
+    func isLevelA() -> Bool {
+        return singing == .A || dancing == .A || acting == .A || visual == .A
+    }
```

Comment:
프로토콜 기본 구현을 사용해서 리팩토링 하셨군요! 👍👍

다만, 이 프로토콜을 채택하는 타입은 노래, 춤, 연기, 비주얼의 재능과 함께 `합격할만한 재능인지 판별하는` 능력도 생기겠네요?
합격자를 선별하는 객체는 `Talent`프로토콜을 채택한 객체일까요? 🤔
이대로라면 지원자가 스스로 합격여부를 체크하는 모습일 것 같아요. 맞는 방향성의 설계인지 살짝 의문이 듭니다!

틀렸다는 뜻은 절대 아닙니다! 가독성은 훨씬 좋아진건 맞으니까요! 따라서 꼭 수정하실 필요도 없습니다.
저였다면 이 체크하는 로직을 `AuditionManager`의 메서드로 넣을거 같긴 해요. 😅😅
한번 어느 방향성이 맞을지 고민해보시기 바랍니다!

추가적으로, 제가 이렇게 구현한다면, 연산 프로퍼티를 활용할 것 같긴 합니다. ㅎㅎ
```suggestion
    var isLevelA: Bool {
        return singing == .A || dancing == .A || acting == .A || visual == .A
    }
```